scripts/docker/docker-node.sh

```diff
@@ -17,17 +18,17 @@ function usage()
 usage: $(basename $0) options blskey blspass
 
 options:
-  -t tag      : tag of the image, default: $tag
-  -p base_port: base port, default: $port_base
-  -n network  : network type
-  -z dns_zone : dns zone
-  -d db_dir   : harmony db directory
+  -t tag                      : tag of the image, default: $tag
+  -p base,sync,rpc,wss        : all port setting, default: $ports
+  -n network                  : network type
```

Comment:
-n is defined here but it's not used in the actual script.

```
$ ./docker-node.sh -t test -n testnet BLS.key blspass.txt
./docker-node.sh: illegal option -- n
```

Reply:
removed -n